torch/testing/_internal/common_utils.py

```diff
@@ -2424,6 +2425,19 @@ def assertGreaterAlmostEqual(self, first, second, places=None, msg=None, delta=N
         msg = self._formatMessage(msg, standardMsg)
         raise self.failureException(msg)
 
+    def assert_is_aten_op(self, onnx_model, operator, overload_name=""):
+        all_aten_nodes = [p for p in onnx_model.graph.node
+                          if p.op_type == "ATen" and p.domain == "org.pytorch.aten"]
+        self.assertTrue(len(all_aten_nodes) > 0)
+
+        for op in all_aten_nodes:
+            attrs = {attr.name: attr.s.decode() for attr in op.attribute}
+            if attrs.get("operator") == operator:
+                break
+
+        self.assertEqual(attrs.get("operator"), operator)
```

Comment:
can operator be None?
If not, then replace with
```suggestion
        self.assertEqual(attrs["operator"], operator)
```